### PR TITLE
Added Test cases for missing scenarios.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Completed the TODO mentioned in the file.

### CODE REVIEW

File: tensorflow/lite/mutable_op_resolver_test.cc [hunk 1]
```diff
@@ -112,8 +122,34 @@ TEST(MutableOpResolverTest, FindCustomOp) {
...
+}
+
+TEST(MutableOpResolverTest, FindCustomName) {
+  MutableOpResolver resolver;
+  resolver.AddCustom("AWESOME", GetDummyRegistration());
+
+  const TfLiteRegistration* found_registration = resolver.FindOp("AWESOME", 1);
+  ASSERT_NE(found_registration, nullptr);
+  EXPECT_EQ(found_registration->builtin_code, BuiltinOperator_CUSTOM);
+  EXPECT_TRUE(found_registration->invoke == DummyInvoke);
```
Comment (jdduke) [inline tensorflow/lite/mutable_op_resolver_test.cc]: EXPECT_EQ?

File: tensorflow/lite/mutable_op_resolver_test.cc [hunk 2]
```diff
@@ -112,8 +122,34 @@ TEST(MutableOpResolverTest, FindCustomOp) {
   EXPECT_EQ(found_registration->builtin_code, BuiltinOperator_CUSTOM);
   EXPECT_TRUE(found_registration->invoke == DummyInvoke);
   EXPECT_EQ(found_registration->version, 1);
-  // TODO(ycling): The `custom_name` in TfLiteRegistration isn't properly
```
Comment (jdduke) [inline tensorflow/lite/mutable_op_resolver_test.cc]: Hmm, this CL isn't really addressing this. I think the idea is to initialize the ".custom_name" field of the registration, and validate that.

File: tensorflow/lite/mutable_op_resolver_test.cc [hunk 3]
```diff
@@ -112,8 +122,34 @@ TEST(MutableOpResolverTest, FindCustomOp) {
...
+
+  const TfLiteRegistration* found_registration = resolver.FindOp("AWESOME", 1);
+  ASSERT_NE(found_registration, nullptr);
+  EXPECT_EQ(found_registration->builtin_code, BuiltinOperator_CUSTOM);
+  EXPECT_TRUE(found_registration->invoke == DummyInvoke);
+  EXPECT_EQ(found_registration->version, 1);
+  EXPECT_EQ(found_registration->custom_name, "AWESOME");
+}
+
+TEST(MutableOpResolverTest, FindFunctionName) {
```
Comment (jdduke) [inline tensorflow/lite/mutable_op_resolver_test.cc]: This isn't really testing anything new.

**Review by jdduke — CHANGES_REQUESTED** [review]

File: tensorflow/lite/mutable_op_resolver_test.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/mutable_op_resolver_test.cc]: @jdduke , thanks for the review, i have updated the code as per your suggestion, kindly check and approve.

Regards
Amit

File: tensorflow/lite/mutable_op_resolver_test.cc (hunk 2 above)
Comment (amitsrivastava78) [reply tensorflow/lite/mutable_op_resolver_test.cc]: @jdduke , got it , kindly check the new TC update, i have followed as per your suggestion.

Regards
Amit

File: tensorflow/lite/mutable_op_resolver_test.cc (hunk 3 above)
Comment (amitsrivastava78) [reply tensorflow/lite/mutable_op_resolver_test.cc]: @jdduke thanks for the comments, i have removed this TC.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@jdduke , thanks for the review, i have updated the code as per your suggestion, kindly check and approve.

Regards
Amit

**Review by jdduke — APPROVED** [review]

**Pull request merged**